core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -35,43 +37,61 @@
 public class ExoPlayerTwoImpl extends PlayerListenersHolder implements Player {
 
     private static final boolean RESET_POSITION = true;
-    private static final boolean RESET_STATE = false;
+    private static final boolean DO_NOT_RESET_STATE = false;
 
     private final SimpleExoPlayer exoPlayer;
     private final MediaSourceFactory mediaSourceFactory;
     private final ExoPlayerForwarder forwarder;
-    private final ExoPlayerTrackSelector trackSelector;
+    private final ExoPlayerAudioTrackSelector trackSelector;
     private final Heart heart;
     private final LoadTimeout loadTimeout;
 
-    private VideoContainer videoContainer = VideoContainer.empty();
+    private VideoContainer videoContainer;
+
     private int videoWidth;
     private int videoHeight;
 
     public static ExoPlayerTwoImpl newInstance(Context context) {
```

Comment:
not for this PR but I'd totally be down for pulling out the `newInstance` to either it's own class or into the player factory